src/shadowbox/server/manager_service.spec.ts

```diff
@@ -278,11 +276,15 @@ describe('ShadowsocksManagerService', () => {
       };
       const next = (error) => {
         expect(error.statusCode).toEqual(400);
-        responseProcessed = true;
-        done();
       };
 
       await service.setPortForNewAccessKeys(noPort, res, next);
+      await service.setPortForNewAccessKeys(
+        // tslint:disable-next-line: no-any
+          (nonNumericPort as any) as {params: {port: number}}, res, next);
+
+      responseProcessed = true;
```

Comment:
Is this used?

Reply:
It's needed to make the test in the afterEach callback to pass.